src/main/java/frc/robot/utilities/TOFSensorSimple.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.utilities;
+
+import com.playingwithfusion.TimeOfFlight;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import java.util.function.BooleanSupplier;
+
+/** Add your docs here. */
+public class TOFSensorSimple {
+  private TimeOfFlight Sensor;
+  private double threshold;
+
+  public TOFSensorSimple(int ID, double threshold) {
+    Sensor = new TimeOfFlight(ID);
+    this.threshold = threshold;
+  }
+
+  public double getDistance() {
+    return Sensor.getRange();
+  }
+
+  public BooleanSupplier isBeamBroke() {
+    return () -> (getDistance() < threshold);
+  }
+
+  public Trigger getBeamTrigger(Command action) {
+    return new Trigger(isBeamBroke());
+  }
```

Comment:
```suggestion
  public boolean isBeamBroke() {
    return getDistance() < threshold;
  }

  public Trigger beamBroken() {
    return new Trigger(this::isBeamBroke);
  }
```

You have the right idea, just the syntax is a bit tricky. Keeping isBeamBroke() returning a boolean makes it most useful to other classes, and the :: syntax can easily make it into a BooleanSupplier to make the Trigger